src/components/AddressSearch/types.ts

```diff
@@ -79,7 +80,7 @@ type AddressSearchProps = {
     predefinedPlaces?: Place[] | null;
 
     /** A map of inputID key names */
-    renamedInputKeys?: RenamedInputKeysProps;
+    renamedInputKeys?: Address;
```

Comment:
Is there a reason to replace the type here?

I find a little confusing to sometimes use the Address type as an address and sometimes as this key map.